model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -16,13 +16,19 @@
 
 from model_analyzer.constants import LOGGER_NAME
 from model_analyzer.triton.model.model_config import ModelConfig
+from typing import NamedTuple
 
 import abc
 import logging
 
 logger = logging.getLogger(LOGGER_NAME)
 
 
+class ModelBatchingConfig(NamedTuple):
```

Comment:
Does this deserve its own file?
Should it be expanded to something more generic (that could also contain things like cpu_only)?